config/default.js

```diff
@@ -0,0 +1,168 @@
+module.exports = {
+	GitterIRCBridges: [
+		{
+			gitter: {
+				token: process.env.NXB_GITTER_TOKEN,
```

Comment:
@0xazure it does indeed fail to start if these are not given. It errors out with either `Gitter Token or Room not specified!` or `Missing argument! You need to specify host, port, name, channel and welcome message!`